### PR TITLE
[1LP][RFR] Fixed playbook method crud operations

### PR DESCRIPTION
**Pull request by ganeshhubale**
- This PR fixes crud operation of ```automate method type - playbook```
- Also changes in the naming of the automate method types for playbook and inline are resolved in this PR
- This PR updates test case with global fixture 
## PRT RUN
{{ pytest: cfme/tests/ansible/test_embedded_ansible_automate.py -k "test_automate_ansible_playbook_method_type_crud or test_automate_ansible_playbook_method_type" --long-running  -vvvvv }}

### CODE REVIEW

**Comment by ganeshhubale** [comment]
@sbulage Please have a look once it gets into RFE state.

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by ganeshhubale** [comment]
This PR is depend on #9203 for enabling embedded ansible server role

File: cfme/automate/explorer/method.py [hunk 1]
```diff
@@ -231,6 +232,8 @@ def read(self):
 
 
 class MethodAddView(AutomateExplorerView):
+    fill_strategy = WaitFillViewStrategy()
+
```
Comment (sbulage) [inline cfme/automate/explorer/method.py]: **Suggestion:**
Please remove extra line here.

File: cfme/tests/ansible/test_embedded_ansible_automate.py [hunk 2]
```diff
@@ -162,14 +114,16 @@ def alert_profile(appliance, alert, full_template_vm_modscope):
     _alert_profile.delete_if_exists()
 
 
-def test_automate_ansible_playbook_method_type_crud(appliance, ansible_repository, domain,
-        namespace, klass):
+@pytest.mark.meta(automates=[1729999, 1740769])
```
Comment (sbulage) [inline cfme/tests/ansible/test_embedded_ansible_automate.py]: **Question:**
I don't think cloned `bugzilla:1740769`  should be a part of `meta marker`, what do you think .?

File: cfme/automate/simulation.py [hunk 3]
```diff
@@ -46,6 +47,11 @@ def simulate(
         'avp': attributes_values,
     })
     view.submit_button.click()
+    wait_for(
+        lambda: view.flash.is_displayed,
+        delay=10,
```
Comment (mshriver) [inline cfme/automate/simulation.py]: Can we lower this delay please? Any reason to be waiting 10s between each check? I'd check every second

**Pull request merged**